src/schedule.js

```diff
@@ -147,7 +147,11 @@ const schedule = (
       newFiles.add(filename)
       return 0
     } else {
-      return duration
+      if (duration === undefined) {
+        return 0.1
+      } else {
+        return duration
+      }
```

Comment:
```suggestion
      return duration || 0
```
Why is `undefined` present in the first place?  Why would `0.1` be preferable to `0`?

Reply:
> Why is undefined present in the first place?

such as this PR: https://github.com/Kong/kong/pull/12242, `spec/02-integration/05-proxy/10-balancer/08-retries_spec.lua` is a new test files,  if `suiteName` is "dbless" here, `duration` would be `undefined` and return.

Why would 0.1 be preferable to 0?

I follow the approach of this PR: https://github.com/Kong/gateway-test-scheduler/pull/10

Reply:
Please rename `newFiles` to `newTests` and use the `suiteName` and the `filename` as key instead of just the file name.  It was a mistake to assume that if a file was added, it was automatically added for all suites.

Reply:
apply